add team replay q2 objectives (#8772)
PostHog/posthog.com: contents/teams/replay/objectives.mdx

### 🎯 Goal 1: People think of PostHog as a mobile solution

* Mobile replay for React Native

> * Lots of content on how to use PostHog on Mobile

Reviewer: It feels like we're light on product / engineering work for this goal unless we're planning to lead the content over marketing & docs.

Maybe this detail is ok to be left to the Mega issue but some ideas of potential line items we could add here:
- 5 customer feedback interviews
- Launch & charge for mobile replay

Reviewer 2: > we're planning to lead the content over marketing & docs.

my goal here would be to pair up with them, provide the technical bits, and help out, but they would help with the writing, making it nice.
@andyvan-ph would that be something your team can help with?

Reviewer 2: I agree, we can add that "Launch & charge for mobile replay" is part of our goal, and since any improvement is done in 2 and soon 3 platforms (hopefully the wrapper approach alone is enough), the work adds up, I also have a couple of weeks OOO this Q.
Most likely we will add ad-hoc work such as PoE in the Mobile SDKs, etc.

> ### 🎯 Goal 2: Error tracking in people's hands

Reviewer: Wonder if we should be somewhat more ambitious with what we target. Given we're off to a good start I think we should aim for a concrete state. "In beta" or "Ready for public release" don't seem wild

Author: ```suggestion
### 🎯 Goal 2: Error tracking in beta
```

> * the top 5 features implemented

Reviewer 2: I think we should be more specific here, what are the top 5 features?

Reviewer: For me they are:
- Grouping
- Stack traces
- SDK support
- Alerting
- Integrated with other products (this one is less important than the other 4 for me)

Author: fine with listing these here (although going to be duplicated with mega issue)
only concern is these are a little bit "techno bits" rather than "user needs" but 🤷

Reviewer 2: We can also rewrite in order of priority with a more user-centric language such as:

- SDK captures errors
- Stack traces are actionable and human-readable
- Similar errors are grouped together
- New errors with high volume trigger the Alerting system
- Integrated with other products such as Session replay

> * the top 5 features implemented

Author: ```suggestion
* the top 5 features implemented
    * Grouping
    * Stack traces
    * SDK support
    * Alerting
    * Integrated with other products (this one is less important than the other 4 for me)
```

Author: fine with listing these here (although going to be duplicated with mega issue)
only concern is these are a little bit "techno bits" rather than "user needs" but 🤷

* real customers using it and giving us feedback

> * integrated with CDP for notifications

Reviewer 2: is notifications the new alerting?

Author: ```suggestion
* integrated with CDP for notifications/alerting
```

i guess they're inter-changeable for me... an alert is a notification you really care about receiving in a timely manner

* integrated with replay for investigation

### Goal 3: Hire two more people into the team

> * as a goal because hiring takes time

Reviewer 2: Wondering if it should be part of the goals, we mentioned during the retro that it should not have been part of the goals because we can't control it anyway, James said the same thing during the planning.
I'm fine either way.

Author: last time round @charlescook-ph mentioned it should be in the goals for any teams hiring

i don't have a _strong_ feeling one way or another - it does use up my time but only maybe a day a month 

if we hire then it'll take at least a week to onboard, and use all of our time to support the new person(s) as they get going

### 🎯 Goal 1: People think of PostHog as a mobile solution

> * Mobile replay for React Native

Author: ```suggestion
* Mobile replay for React Native
* Launch & charge for mobile replay
```

* Lots of content on how to use PostHog on Mobile

### 🎯 Goal 2: Error tracking in beta

* the top 5 features implemented

>     * Grouping
>     * Stack traces
>     * SDK support
>     * Alerting
>     * Integrated with other products (this one is less important than the other 4 for me)

Reviewer 2: ```suggestion
    * SDK captures errors
    * Stack traces are actionable and human-readable
    * Similar errors are grouped together
    * High-volume errors trigger the Alerting system
    * Integrated with other products such as Session replay
```
